src/datasources/cache/__tests__/fake.cache.service.ts

```diff
@@ -11,8 +11,9 @@ export class FakeCacheService implements ICacheService {
     this.cache = {};
   }
 
-  delete(key: string) {
+  delete(key: string): Promise<number> {
     delete this.cache[key];
+    return Promise.resolve(1);
```

Comment:
`number` represents the number of entries deleted by this operation. We assume `1` for delete operations in `FakeCacheService` as a key should be unique.